packages/vite-plugin-tanstack-start/src/main.ts

```diff
@@ -0,0 +1,25 @@
+import { type Features } from '@netlify/dev'
```

Comment:
oops, I don't think I have a dep on this. shouldn't be necessary anyway with a small change below.

Reply:
Is this comment outdated? I see all the imports here are in deps:
https://github.com/netlify/primitives/blob/7393ea17dc06cd0735d7011407b3c8c5d34ee79b/packages/vite-plugin-tanstack-start/package.json#L47-L50

Tho would be great if we wouldn't need extra dep on `@netlify/dev` to get `Features` type and instead could use something from `@netlify/vite-plugin` as there is always opportunity that this will create duplicate `@netlify/dev` somehow (but this could be follow up)

Reply:
fixed in e1f0722